src/main/java/com/bobocode/svydovets/bibernate/action/ActionQueue.java

```diff
@@ -29,12 +40,14 @@ public void addAction(EntityKey<?> entityKey, Action action) {
                 });
     }
 
+    /** Executes all actions in the queue in the correct order (insert, update, remove). */
```

Comment:
if it matters let's rename better the method to something like executeAllWithOrdering